command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 10:03:15AM -0700, Mike Brown wrote:
> When the Solaris config was put together no one stated that all OCI
> runtimes must include support for solaris configs for testing
> purposes. This as you have stated for the CLI spec. Your insistence
> on using MUST language forces the reader to ask, ok who MUST. You
> are currently saying the runtime MUST. And you are indicating
> through these discussion that the runtime MUST provide a CLI for
> testing purposes.

The proposed [1] wording is [2]:

  In order to be [certified][], compliant runtimes MUST implement at
  least one of the following APIs:

Does anyone have a problem with that?  I thought everybody agreed that
certification tools would stick to OCI-defined APIs [3].  Maybe if I
drop “compliant”, since that's already covered by [4]?

> I do not believe the document will be harmed by being at tad more
> explicit.

My concern is that if “a runtime can be compliant with subsets of
runtime-spec and still be OCI certified” is contentious at all, it's a
systemic problem with runtime-spec, and not specific to this command
line API file/portion.  I don't like case-by-case solutions to
systemic problems.  I don't think a systemic problem exists [5], but
if there is not consensus around whether it does or doesn't, I'd like
to see a solution that *also* covers runtimes that leave out
config-solaris.md, or config-linux.md, etc., and not just a solution
that covers runtimes that leave out command-line-interface.md

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72713742
[2]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72701237
[3]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72709539
[4]: https://github.com/opencontainers/runtime-spec/tree/v1.0.0-rc1#open-container-runtime-specification
[5]: https://github.com/opencontainers/runtime-spec/pull/513/files#r72820802